### PR TITLE
add unstable `is_sorted` feature

### PR DESCRIPTION
**Pull request by robjtede**

### CODE REVIEW

File: data/unstable/is_sorted.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+title = "`is_sorted_by` / `is_sorted_by_key` methods on iterables"
+flag = "is_sorted"
+rfc_id = 2351
+tracking_issue_id = 53485
+aliases = ["is_sorted_by", "is_sorted_by_key"]
```
Comment (jplatte) [inline data/unstable/is_sorted.toml:5]: Aliases only make sense if they're not part of the title already.

**Review by jplatte — CHANGES_REQUESTED** [review]
This only mentions `is_sorted_by` and `is_sorted_by_key`, not the "main" `is_sorted`. Also I think it would make sense to split this into at least two features (one for the slice methods, one for the iterator methods) because "iterables" would probably include `LinkedList`, which does not have these methods.

File: data/unstable/is_sorted.toml (hunk 1 above)
Comment (jplatte) [reply data/unstable/is_sorted.toml:5]: Could shorten the title and add `items` though (these are actually shown on the feature, in contrast to `aliases`).

Comment (robjtede) [reply data/unstable/is_sorted.toml]: > This only mentions is_sorted_by and is_sorted_by_key, not the "main" is_sorted.

coulda sworn that `is_sorted` was stablized... ahh well

> split this into at least two features 

yeah makes sense

**Pull request merged**